execution/evm/vm.go

```diff
@@ -936,6 +945,19 @@ func (vm *VM) call(caller, callee acm.MutableAccount, code, input []byte, value
 			vm.Debugf(" => [%v, %v] (%d) 0x%X\n", offset, size, len(output), output)
 			return output, nil
 
+		case REVERT: // 0xFD
+			offset, size := stack.Pop64(), stack.Pop64()
+			output, memErr := memory.Read(offset, size)
+			if memErr != nil {
+				vm.Debugf(" => Memory err: %s", memErr)
+				return nil, firstErr(err, ErrMemoryOutOfBounds)
+			}
+
+			outputMessage := output[offset:size]
```

Comment:
This is unnecessary, remove and just return `output` which is the correct slice already (as in `RETURN`)